valkey.conf

```diff
@@ -1276,24 +1276,24 @@ acllog-max-len 128
 # in order to instead release memory in a non-blocking way like if UNLINK
 # was called, using the following configuration directives.
 
-lazyfree-lazy-eviction no
-lazyfree-lazy-expire no
```

Comment:
The large comment above these configs needs to be rewritten. It contains things like

```
# The server has two primitives to delete keys. One is called DEL and is a blocking
# deletion of the object. It means that the server stops processing new commands
# in order to reclaim all the memory associated with an object in a synchronous
# way.
```

and

```
# In all the above cases the default is to delete objects in a blocking way,
# like if DEL was called. However you can configure each case specifically
# in order to instead release memory in a non-blocking way like if UNLINK
# was called, using the following configuration directives.
```

We should instead mention things like what you wrote in the PR description:

> There are many problems with running flush synchronously. Even in
single CPU environments, the thread managers should balance between
the freeing and serving incoming requests.

Reply:
right, i guess we do neet do update the configuration comments, i am bad at english writing, would you be able to draft some wording for me?

Reply:
Yes, I can try to write something.

We should change even the first sentence "The server has two primitives to delete keys. One is called DEL and is a blocking deletion of the object." Even this is not true anymore, by default.